mixserv/src/main/java/hivemall/mix/server/MixServer.java

```diff
@@ -94,6 +102,20 @@ static Options getOptions() {
         return opts;
     }
 
+    @Override
+    public String toString() {
+        final List<String> configs = new ArrayList<String>();
```

Comment:
Please avoid using StringUtils#join and use StringBuilder.
IntelliJ and Eclipse have auto generation methods of toString(). "aaa"+"bbb" is compiled into `sb.append("aaa").append("bbb")`.

```
    @Override
    public String toString() {
        return "MixServer [port=" + port + ", numWorkers=" + numWorkers + ", ssl=" + ssl
                + ", scale=" + scale + ", syncThreshold=" + syncThreshold + ", sessionTTLinSec="
                + sessionTTLinSec + ", sweepIntervalInSec=" + sweepIntervalInSec + ", jmx=" + jmx
                + ", state=" + state + "]";
    }
```